docs/docs/examples/workflow/router_query_engine.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #113.                if isinstance(response, (StreamingResponse, PydanticResponse)):
</pre><p>Will probably need to check for AsyncStreamingResponse as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>